tasks/cephfs/test_volume_client.py

```diff
@@ -452,3 +452,54 @@ def test_evict_client(self):
 
         # We must hard-umount the one that we evicted
         self.mounts[2].umount_wait(force=True)
+
+    def test_purge(self):
+        """
+        Reproducer for #15266, exception trying to purge volumes that
```

Comment:
both the volume name and files within the volume need to be non-ascii to reproduce the error, right?